plugin/core/sessions.py

```diff
@@ -605,6 +605,9 @@ def get_language_id(self) -> Optional[str]:
     def get_view_in_group(self, group: int) -> sublime.View:
         ...
 
+    def is_self(self, buffer_id: int) -> bool:
+        ...
```

Comment:
Remove unnecessary changes and fix linter issues.

Reply:
👌🏿